### ISSUE TITLE
Displays do not show 3rd Party hosted images in a Presentation

### ISSUE THREAD

**Issue by robb-price** [Rise-Vision/rise-launcher-electron#831]
**Describe the bug**
When showing a third party hosted image in the Image Widget on the Linux Player, it never shows, and only keeps showing "File is Downloading" or "Please wait while your file is downloaded"

**Reach**
Only 1 user has brought this up, but it has the potential to affect all Linux users.

**Impact**
The workaround is to set the background of the Placeholder to be that image. In that case, it will show.

**To Reproduce**
1. Use a third party hosted image like this: https://www.w3schools.com/w3css/img_lights.jpg in the Image Widget in a Presentation.
2. Play on a Linux Player.
3. Notice it never shows the image

**Expected behavior**
Similar to Windows, ChromeOS and Pi, it should show the image.

**Desktop (please complete the following information):**
 - OS: Linux

**Comment by PatrickA-RiseVision** [comment]
I tested this on my player both before and after an update based on customer reports that video widget was also exhibiting this behavior.

I built a simple presentation with a single video widget and single image widget.

In 2019.10.29.21.57, I was not able to replicate the issue with either the video widget or image widget, whether I was using storage, linking files in storage, or linking external files.

In RisePlayerElectron 2019.11.21.20.53 Licensed, I can replicate the issue with both the video widget and image widget when linking external files.  I can provide screenconnect to my player privately.

I wasn't able to replicate the issue with files in storage, whether I was picking them in-app or linking them.
